ppr-ui/src/components/mhrTransfers/DeathCertificate.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div class="mhr-death-certificate">
```

Comment:
Might want to make it an id. 
Also, i'm not entirely sure we have to label it with mhr. 🤔